cli/app.go

```diff
@@ -299,15 +299,10 @@ var app = &cli.App{
 					Action: DataExportAction,
 				},
 				{
-					Name:      "delete",
+					Name:      "delete-binary",
 					Usage:     "delete binary data from Viam cloud",
-					UsageText: fmt.Sprintf("viam data delete <%s> [other options]", dataFlagDataType),
```

Comment:
Just a side note that we could alternatively have subcommands for `binary` and `tabular`, under delete, which is slightly cleaner IMO but this is fine too.

Reply:
Very very good suggestion, updated!